ivy_tests/test_ivy/test_frontends/test_numpy/test_fft/test_discrete_fourier_transform.py

```diff
@@ -29,3 +31,33 @@ def test_numpy_iftt(
         axis=-1,
         norm=None
     )
+
+
+@handle_frontend_test(
+    fn_tree="numpy.fft.ifft2",
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("numeric"),
+        available_shapes=[(4, 4), (3, 5), (2, 2)]
+    )
+)
+def test_numpy_ifft2(
+        dtype_and_x,
+        frontend,
+        test_flags,
+        fn_tree,
+        on_device
+):
+    input_dtype, x = dtype_and_x
+    a = ivy.array(x, dtype=ivy.complex128)
+
+    # Test `ifft2` function with different parameter values
+    for n in [None, 4]:
+        for axis in [(-1, -2), (-2, -1)]:
+            for norm in [None, "forward", "backward"]:
+
+                np_output = np.fft.ifft2(x, n=n, axes=axis, norm=norm)
```

Comment:
this module shouldn't be needed here, whether it's native numpy or ivy backend numpy

Reply:
you should just directly use the `test_frontend_function`